src/components/Authentication/ForgotPasswordForm.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from "@/components/ui/button";
+import { Card, CardContent, CardDescription, CardFooter, CardHeader, CardTitle } from "@/components/ui/card";
+import { Input } from "@/components/ui/input";
+import { useForm } from "react-hook-form";
+import { useState } from "react";
+import * as z from "zod";
+import { Form, FormControl, FormField, FormItem, FormLabel, FormMessage } from "../ui/form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import toast from "react-hot-toast";
```

Comment:
Package not install.
Use [sonner](https://ui.shadcn.com/docs/components/sonner) instead.